core/services/pipeline/common_test.go

```diff
@@ -323,3 +324,66 @@ func TestTaskRunResult_IsPending(t *testing.T) {
 	trrWithFinishedAt := &pipeline.TaskRunResult{FinishedAt: null.NewTime(time.Now(), true)}
 	assert.False(t, trrWithFinishedAt.IsPending())
 }
+
+func TestSelectGasLimit(t *testing.T) {
+	t.Parallel()
+
+	gcfg := cltest.NewTestGeneralConfig(t)
+	gcfg.Overrides.GlobalEvmGasLimitDefault = null.IntFrom(999)
+	gcfg.Overrides.GlobalEvmGasLimitDRJobType = null.IntFrom(100)
+	gcfg.Overrides.GlobalEvmGasLimitVRFJobType = null.IntFrom(101)
+	gcfg.Overrides.GlobalEvmGasLimitFMJobType = null.IntFrom(102)
+	gcfg.Overrides.GlobalEvmGasLimitOCRJobType = null.IntFrom(103)
+	gcfg.Overrides.GlobalEvmGasLimitKeeperJobType = null.IntFrom(103)
+	cfg := evmtest.NewChainScopedConfig(t, gcfg)
+
+	t.Run("spec defined gas limit", func(t *testing.T) {
+		t.Parallel()
```

Comment:
The overhead on making these parallel is probably overkill considering how trivial the test cases are

Reply:
Ah I missed that point. Thank you!